recipes/arraymorph/meta.yaml

```
@@ -0,0 +1,72 @@
+package:
+  name: arraymorph
```

```suggestion
  name: libarraymorph
```
Our naming convention is that C libraries use the lib prefix for the package name unless they are a package that also contains programs with CLI or GUI entry points.